app/web/src/features/auth/signup/Signup.tsx

```diff
@@ -178,66 +197,52 @@ export default function Signup() {
     <>
       {authenticated && <Redirect to="/" />}
       <HtmlMeta title={SIGN_UP} />
-
-      {/***** MOBILE ******/}
-      <Hidden mdUp>
-        <div className={authClasses.page}>
+      <div
+        className={classNames(
+          authClasses.page,
+          classes.stickyPage,
+          authClasses.pageBackground
+        )}
+      >
+        <header className={authClasses.header}>
+          <div className={authClasses.logoContainer}>
+            <CouchersLogo />
+            <div className={authClasses.logo}>{COUCHERS}</div>
+          </div>
+        </header>
+        <div className={authClasses.content}>
+          <div className={authClasses.introduction}>
+            <Typography
+              classes={{ root: authClasses.title }}
+              variant="h1"
+              component="span"
+            >
+              {INTRODUCTION_TITLE}
+            </Typography>
+            <Typography
+              classes={{ root: authClasses.subtitle }}
+              variant="h2"
+              component="span"
+            >
+              {INTRODUCTION_SUBTITLE}
+              <Divider className={authClasses.underline}></Divider>
+            </Typography>
+          </div>
+          <div style={{ width: "45%" }}></div>
```

Comment:
I have to say this filler `div` seems a bit hacky. How come we need this here but not for the login page? It's the same layout as far as I can see...

Reply:
Yes, the layout is a bit complicated in order to scroll the tall signup form only but keep the scrollbar on the root page.
To do this, the page is in two parts: the header + left content container, which is `fixed`, and the form, which scrolls, and they have to be siblings. So this hacky div is to match the layout on the login page, where the left and right content are siblings in a flex.